docs/book/driver-deployment/installation.md

```diff
@@ -0,0 +1,226 @@
+clusterrole.rbac.authorization.k8s.io/vsphere-csi-controller-role created
+clusterrolebinding.rbac.authorization.k8s.io/vsphere-csi-controller-binding created
+```
+
+Click [here](https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/manifests/vsphere-7.0/vanilla/rbac) to view the roles assigned to vSphere CSI Driver.
+
+## Install vSphere CSI driver
+
+Our CSI Controller runs as a Kubernetes deployment, with a replica count of 1. For version `v2.0.0`, the `vsphere-csi-controller` Pod consists of 6 containers – the CSI controller, External Provisioner, External Attacher, External Resizer, Liveness probe and vSphere Syncer. The vSphere Syncer introduces features supported by our [Cloud Native Storage solution](https://docs.vmware.com/en/VMware-vSphere/6.7/Cloud-Native-Storage/GUID-51D308C7-ECFE-4C04-AD56-64B6E00A6548.html)
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/manifests/vsphere-7.0/vanilla/deploy/vsphere-csi-controller-deployment.yaml
+```
+
+There is also a CSI node Daemonset to be deployed, that will run on every node.
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/manifests/vsphere-7.0/vanilla/deploy/vsphere-csi-node-ds.yaml
+```
+
+Click [here](https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/manifests/vsphere-7.0/vanilla/deploy) to view the deployment manifest for vSphere CSI driver. 
+
+NOTE: Visit [vSphere CSI Driver - Deployment with Topology](https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/docs/book/driver-deployment/deploying_csi_with_zones.md) to deploy your kubernetes cluster with topology aware provisioning feature.
+
+## Verify that CSI has been successfully deployed
+
+To verify that the CSI driver has been successfully deployed, you should observe that there is one instance of the vsphere-csi-controller running on the master node and that an instance of the vsphere-csi-node is running on each of the worker nodes.
+
+```bash
+$ kubectl get deployment --namespace=kube-system
+NAME                          READY   AGE
+vsphere-csi-controller        1/1     2m58s
+kubectl get daemonsets vsphere-csi-node --namespace=kube-system
+NAME               DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+vsphere-csi-node   4         4         4       4            4           <none>          3m51s
+```
+
+Verify that the vSphere CSI driver has been registered with Kubernetes
+
+```bash
+$ kubectl get csidrivers
+NAME                           CREATED AT
+csi.vsphere.vmware.com   2020-04-13T12:29:11Z
+
+```
+
+Verify that the CSINodes have been created
+
+```bash
+$ kubectl get CSINode
+NAME                 CREATED AT
+<k8s-worker1-name>   2020-04-13T12:30:29Z
+<k8s-worker2-name>   2020-04-13T12:30:38Z
+<k8s-worker3-name>   2020-04-13T12:30:21Z
+<k8s-worker4-name>   2020-04-13T12:30:26Z
+```
+
+Verify ProviderID has been added to all nodes
```

Comment:
Can we put a note, that `ProviderID` will be set by CPI and not by CSI driver?  otherwise this will create confusion.

Reply:
moved it to prerequisite section